pkg/kubelet/config/file_linux.go

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright 2014 The Kubernetes Authors All rights reserved.
```

Comment:
Header should be updated according to this [boilerplate](https://github.com/kubernetes/kubernetes/blob/master/hack/boilerplate/boilerplate.go.txt).